django/thunderstore/core/settings.py

```diff
@@ -86,6 +86,9 @@
 
 
 def load_db_certs():
+    if not any((DB_CLIENT_CERT, DB_CLIENT_KEY, DB_SERVER_CA)):
+        return
+
     cert_dir = "/etc/ssl/private/db-certs/"
     if not os.path.exists(cert_dir):
         os.makedirs(cert_dir)
```

Comment:
Add a try-catch here instead. The cert mapping already works even if the variables are defined, but what you are failing on is creating this directory.

With the current change all that happens is if someone tries to use certs in their local environment, it will crash, so in other words we've just pushed the problem down the line.

A real solution would be to
1. Read `cert_dir` be read from environment variables
2. Set `/etc/ssl/private/db-certs/` as the default parameter of that environment variable when building a docker image
3. Check if `cert_dir` environment variable has been defined, and if not, exit the function